content/home/src/howto_guides/submit2nmdc.md

```diff
@@ -110,11 +128,99 @@ A ‘Jump to column’ feature is available for searching for specific metadata
-)
+After selecting the validate button, cells will become color coded to indicate invalid and incomplete metadata. Dark pink cells indicate a required cell has been left empty. Light pink indicates that there is an error in the formatting of the information entered into that cell. The ‘Column Information’ tool, described above, provides expected structure patterns and examples of valid metadata in order to help remedy errors and invalid inputs.
+
+## Field Notes Functionality
+
+https://microbiomedata.org/field-notes/
+
+The NMDC Field Notes App is for mobile device use and is available for beta testing. See (TODO) to sign up!
+
+This app is designed to streamline real-time registration and collection of standardized metadata and field measurements. 
+
+Each section of the app will automatically launch a quick walk-through. You can press the → button to continue the tour or press the X button to end it. To re-initiate the walk through at any time, visit the Settings page and select ‘App Walk-Through’
+
+Available in the app is general fieldwork guidance and recommendations for collecting proper metadata for your sample type(s) to help you plan your fieldwork and metadata collection. We recommend viewing this information before you go out into the field.
+
+TODO: Add screen shot images
+
+The Field Notes App requires ORCiD authentication to access. 
+
+Submissions via Field Notes are available in the Submission Portal as an automatic backup when connected to wifi. The app is available for offline use, but requires internet connection to **start** a submission. 
+
+
+TODO: Update all these to field notes!
+
+(Step 1 of 3) Welcome to the NMDC Field Notes mobile app! 
+
+[![](../_static/images/howto_guides/portal_guide/Create_submission.png)](../_static/images/howto_guides/portal_guide/Create_submission.png)
+
+Once signed in with an ORCiD, you will have the ability to access existing studys or create a study using the 'New’ button. Details about each section are outlined below. Users can also return to existing submissions saved under the ORCiD account to resume their work at any point. 
+
+### New Study
+
+[![](../_static/images/howto_guides/portal_guide/study_info.png)](../_static/images/howto_guides/portal_guide/study_info.png)
+
+The Study Information page requires a valid ‘Study Name’ and a valid email address. All other fields are optional and can be completed in the app or in the Submission Portal. Study names and all other information can be updated or added via the app or Submission Portal. See [header](#submission-portal-functionality) [header](#study) for more details about Study information.
+
+Also in the New Study you will select your "Environment Template". This will correspond to the GSC Environmental Extension. More information is available above in sections [header](#environmental-package). Multiple environments can be selects and submitted in one submission.
```

Comment:
"the GSC Environmental Extension." -> "the GSC Environmental Extension that best fits your samples." ?
"can be selects" -> "can be selected"

Reply:
```suggestion
Also in the New Study you will select the "Environment Template" that best fits your samples. The templates available correspond to the GSC Environmental Extensions. More information is available above in sections [header](#environmental-package). Multiple environments can be selected and submitted in one submission.
```

Reply:
@simroux thoughts on that change?